### ISSUE TITLE
-ash: service: not found

### ISSUE THREAD

**Issue by BeardyBrah** [porech/roborock-oucher#19]
G'day from Australia,
Not sure if a genuine issue, or user error. 

I've used WinSCP to transfer all the files across to the directories as in the readme, swapping over to PuTTY and each time I'm trying to write 'service oucher stop' or 'service oucher restart' I get the message '-ash: oucher: not found'.

same is true for another command, 'chmod +x /etc/init/S12oucher' returns 'chmod: cannot access '/etc/init/S12oucher': No such file or directory'

up to this stage I havn't been able to get oucher to work at all :(
I'm pretty new to PuTTY & WinSCP so its probobly a user-error on my end, but would love any assistance!

**Comment by BeardyBrah** [comment]
Thanks so much for your reply.
Ok I've double checked as per your instructions.

I've got /usr/local/bin/oucher with rwxr-xr-x rights added in WinSCP. 
Attempted the command chmod +x /usr/local/bin/oucher in PuTTY and it seems to have taken.. strangely. no erros this time.
I've did indeed have S12oucher in /etc/init (now removed)
I've just sent the reboot command and will test.

The robot I'm using is the Roborock S5 (non-max version) with Valetudo 002034
